tests/__snapshots__/recommended.js.snap

```diff
@@ -4,7 +4,6 @@ exports[`recommended rules 1`] = `
 Array [
```

Comment:
I'm not sure what this file is, but I haven't used jest before. Do we want this committed? Just double checking.

Reply:
yes